KinEcosystem/ViewModel/OrderViewModel.swift

```diff
@@ -50,16 +58,28 @@ class OrderViewModel {
             details = ""
         }
         color = indicatorColor
+        
         title = model.title.attributed(18.0, weight: .regular, color: titleColor) +
                 details.attributed(14.0, weight: .regular, color: detailsColor)
         var subtitleString = model.description_
         if let shortDate = Iso8601DateFormatter.shortString(from: model.completion_date as Date) {
             subtitleString = subtitleString + " - " + shortDate
         }
+        
         subtitle = subtitleString.attributed(14.0, weight: .regular, color: .kinBlueGreyTwo)
         
-        amount = ((model.offerType == .earn ? "+" : "-") + "\(Decimal(model.amount).currencyString()) ").attributed(16.0, weight: .medium, color: .kinBlueGreyTwo)
-
+        var amountOperator = ""
+        switch model.offerType {
+        case .earn:
+            amountOperator = "+"
+        case .pay_to_user:
+            if let address = selfPublicAddress {
+                amountOperator = address == model.blockchain_data?.recipient_address ? "+" : "-"
+            }
+        default: // .spend
+            amountOperator = "-"
+        }
         
+        amount = ((amountOperator) + "\(Decimal(model.amount).currencyString()) ").attributed(16.0, weight: .medium, color: .kinBlueGreyTwo)
```

Comment:
not related to this PR, just wondering, but seems like ios UI is different than android... (different color for earn/spend, dot color)

Reply:
We definitely need to address the UI differences, as you said, not in this PR scope.